control-plane/connect-inject/constants/constants.go

```diff
@@ -72,6 +72,9 @@ const (
 	ConsulKubernetesCheckName = "Kubernetes Readiness Check"
 
 	KubernetesSuccessReasonMsg = "Kubernetes health checks passing"
+
+	// ProxyIDVolumeName is the name of the volume that contains the proxy ID.
+	ProxyIDVolumeName = "/consul/mesh-inject"
```

Comment:
This is technically the volume _path_, not the name I think. Does that make sense?

Reply:
yuup! updated this to reflect that